### PR TITLE
`Token` isExAnte non-nullable for all `Token` entites

### PR DESCRIPTION
**Pull request by psparacino**
This will keep things consistent and reduce complexity in the api.

### CODE REVIEW

**Review by 0xAurelius — APPROVED** [review]

File: polygon-digital-carbon/subgraph.template.yaml [hunk 1]
```diff
@@ -3,11 +3,6 @@ description: Polygon Carbon
 repository: https://github.com/KlimaDAO/klima-subgraph
 schema:
   file: ./schema.graphql
-features:
-  - grafting
-graft:
-  base: QmUTK4dB7CXVvN2XsX1temzZGcryQ59j8YKdQP5Z2AwzVZ
-  block: 58282296
```
Comment (0xAurelius) [inline polygon-digital-carbon/subgraph.template.yaml:10]: This needs to be rebased, as of the other PR we now have a commented out `graft` section

File: polygon-digital-carbon/src/utils/Token.ts [hunk 2]
```diff
@@ -66,3 +70,33 @@ export function createICRTokenWithCall(tokenAddress: Address, tokenId: BigInt):
...
+
+    let decimalCall = tokenContract.try_decimals()
+    if (decimalCall.reverted) token.decimals = 18 // Default to 18 decimals
+    else token.decimals = decimalCall.value
+
+    token.latestPriceUSD = tokenAddress == USDC_ERC20_CONTRACT ? BigDecimal.fromString('1') : ZERO_BD
+    token.latestPriceUSDUpdated = ZERO_BI
+    token.latestPricePerKLIMA = ZERO_BD
+    token.latestPricePerKLIMAUpdated = ZERO_BI
+    token.isExAnte = false
```
Comment (0xAurelius) [inline polygon-digital-carbon/src/utils/Token.ts:97]: So right now no tokens will ever be labeled exAnte? We add that logic later?

File: polygon-digital-carbon/src/utils/CarbonPool.ts [hunk 3]
```diff
@@ -3,7 +3,7 @@ import { CarbonPool, CarbonPoolDailySnapshot, PoolDeposit, PoolRedeem } from '..
 import { ZERO_BI } from '../../../lib/utils/Decimals'
 import { dayFromTimestamp } from '../../../lib/utils/Dates'
 import { loadOrCreateCarbonPoolCreditBalanceDailySnapshot } from './CarbonPoolCreditBalance'
-import { loadOrCreateToken } from '../../../lib/utils/Token'
```
Comment (0xAurelius) [inline polygon-digital-carbon/src/utils/CarbonPool.ts:6]: Is this old function still being used somewhere else? If not it should prob be deleted...

**Review by 0xAurelius — CHANGES_REQUESTED** [review]
Needs to be rebased and one question about unused function

File: polygon-digital-carbon/src/utils/Token.ts (hunk 2 above)
Comment (psparacino) [reply polygon-digital-carbon/src/utils/Token.ts:97]: This function is only used for loadOrCreateCarbonPool, ICR has its own token creation function `createICRTokenWithCall` that will assign the correct exPost or exAnte

File: polygon-digital-carbon/src/utils/CarbonPool.ts (hunk 3 above)
Comment (psparacino) [reply polygon-digital-carbon/src/utils/CarbonPool.ts:6]: @cujowolf 's suggestion I moved the function inside digital-carbon as there were issues with it being in lib. Used below in `loadOrCreateCarbonPool`

Comment (0xAurelius) [reply polygon-digital-carbon/src/utils/CarbonPool.ts:6]: Right I see that but is the old version in `lib` still there/still being used? Don't see it being removed in this PR

Comment (psparacino) [reply polygon-digital-carbon/src/utils/CarbonPool.ts:6]: Ah ok, removed

File: polygon-digital-carbon/subgraph.template.yaml (hunk 1 above)
Comment (psparacino) [reply polygon-digital-carbon/subgraph.template.yaml:10]: The rebase was up to date but I reset to the commit I removed the graft and commented it out there instead vs adding more commits to add + comment out

**Review by cujowolf — APPROVED** [review]

**Review by 0xAurelius — APPROVED** [review]

**Pull request merged**